standard/types.md

```diff
@@ -54,7 +54,8 @@ interface_type
     ;
 
 array_type
-    : non_array_type rank_specifier+
+    | non_array_type rank_specifier+
+    | non_array_type ( nullable_type_annotation rank_specifier+ )+
```

Comment:
Does this allow `int??[]`?

Reply:
No, because a nullable type isn't a `non_array_type`.

Reply:
If I'm reading it right, `non_array_type` includes `value_type` which includes `nullable_value_type` which I believe `int?` would be.